frontend/package.json

```diff
@@ -1,79 +1,75 @@
 {
 	"name": "frontend",
-	"version": "1.0.0",
+	"version": "0.0.0",
 	"private": true,
-	"dependencies":
-	{
-		"@testing-library/cypress": "^8.0.2",
-		"@types/enzyme": "^3.10.11",
-		"axios": "^0.26.1",
-		"enzyme": "^3.11.0",
-		"react": "^17.0.2",
-		"react-dom": "^17.0.2",
-		"react-scripts": "5.0.0",
-		"typescript": "4.6.2",
+	"dependencies": {
+		"@angular/animations": "~13.3.0",
+		"@angular/common": "~13.3.0",
+		"@angular/compiler": "~13.3.0",
+		"@angular/core": "~13.3.0",
+		"@angular/forms": "~13.3.0",
+		"@angular/platform-browser": "~13.3.0",
+		"@angular/platform-browser-dynamic": "~13.3.0",
+		"@angular/router": "~13.3.0",
+		"@isnotdefined/eslint-config": "4.4.0",
```

Comment:
Update to eslint-config to 4.5.1

Reply:
It is update to 4.6.0 and shows me that there is 6.7.0?